flink-connector/src/main/java/com/datastrato/gravitino/flink/connector/catalog/BaseCatalog.java

```diff
@@ -337,6 +411,29 @@ public void alterPartitionColumnStatistics(
 
   protected abstract PropertiesConverter getPropertiesConverter();
 
+  protected CatalogBaseTable toFlinkTable(Table table) {
+
+    org.apache.flink.table.api.Schema.Builder builder =
+        org.apache.flink.table.api.Schema.newBuilder();
+    for (Column column : table.columns()) {
+      builder
+          .column(column.name(), TypeUtils.toFlinkType(column.dataType()))
```

Comment:
handle nullable?

Reply:
@FANNG1 Where may throw a nullable exception？ Can you give me more details? Thanks.

Reply:
OH, it's `column nullable` not `nullable exception`